.eslintrc

```diff
@@ -2,7 +2,7 @@
   "extends": "@dabh/eslint-config-populist",
   "rules": {
     "one-var": ["error", { "var": "never", "let": "never", "const": "never" }],
-    "linebreak-style": ["error", "windows"],
+    "linebreak-style": ["warn", "windows"],
```

Comment:
since this is a departure from the current linebreaks used throughout I feel we should just convert the editorconfig/eslintrc to enforce unix linebreaks. 

Curious why we even want to enforce from a linting perspective anyhow I don't have any preference here so maybe i'm just uninformed but git tends to account for line endings quite well with the default settings. If this is something we really want to do I feel this is better suited for configuring a `.gitattributes` file with the following contents for the repository:

```* text eol=lf```

Reply:
I don't think we should have either errors _or_ warnings about linebreak style.  I agree with @Maverick1872 that we should do what git likes to do by default, which is to check in all files as LF but check out as CRLF on Windows.

Reply:
Looks good for me, I just wanted to have eslint synch with existing sources but seems to be more a problem than a solution...